### ISSUE TITLE
thanks for making this plugin. Is it possible to fix angular attributes?

### ISSUE THREAD

**Issue by badpenguin** [hexaviewtech/ionic-netbeans-plugin#2]
For instance in the same template:

1) that's OK, angularJS is working

<div ng-hide="is_loading"></div>

2) there is a warning about unknown attribute "ng-hide" from angular
<ion-list ng-hide="is_loading">

**Comment by sampatsinghkhatri** [comment]
I think it is the issue with the angular-plugin version used  , there is two or three  versions of angular plugin are available so try other one is one option and on the other hand there is the capability in netbeans ide to add your own custom tags , so second one is temporary fix the issue for development ide .

**Comment by badpenguin** [comment]
Can you point me out to some DOC about how i can contribute to this plugin adding the missing attributes? I was unable to find it on Netbean's website :-(

**Comment by Chris2011** [comment]
angular plugin? In netbeans or what? There is native support for angular in netbeans, No plugin needed.

**Comment by badpenguin** [comment]
Something wrong then. I'm using both 8.1 and 8.2dev and i got warning for ng-view and other ng-\* attributes.

**Comment by Chris2011** [comment]
I think this is normal, because ion elements doesn't know ng attributes. Only my opinion. If you hit Alt + Enter you can add ng-view for example to the public project settings.

**Comment by Chris2011** [comment]
Can you add it there and can you give feedback whether this helped or not?

**Comment by badpenguin** [comment]
Its not a problem about auto-completion. Its a syntax warning :)

**Comment by Chris2011** [comment]
Yes I know alt + enter is not code completion as far as you set it not to this hotkey. Alt + Enter is the hint functionality where you find informations as you can see it here: 
![syntax-error-html](https://cloud.githubusercontent.com/assets/795658/14243697/488e78fe-fa57-11e5-8f27-0759f77a2879.png)

This is your problem right?

**Comment by badpenguin** [comment]
Yes, i know i can have a local file with exceptions, but would be great to have it in the plugin.
